jerry-port/default/default-date.c

```diff
@@ -65,12 +75,11 @@ double jerry_port_get_local_time_zone_adjustment (double unix_ms,  /**< ms since
 #else /* !HAVE_TM_GMTOFF */
   (void) unix_ms;
   (void) is_utc;
-#ifdef _WINDOWS
+#ifdef _WIN32
   FILETIME fileTime, localFileTime;
   SYSTEMTIME systemTime, localSystemTime;
   ULARGE_INTEGER time, localTime;
 
-  _tzset ();
   UnixTimeToFileTime ((LONGLONG) (unix_ms / 1000), &fileTime);
```

Comment:
To unify computations, shouldn't this function be also updated a bit? I.e., to have it as `UnixTimeMsToFileTime`, so that division is unnecessary here, and the body of the function becomes actually a mathematical inverse of `FileTimeToUnixTimeMs`.

Reply:
Good point. I already unified and simplified them (casts, magic numbers, ...)